src/main/java/net/dries007/tfc/objects/fluids/FluidsTFC.java

```diff
@@ -64,6 +64,23 @@
     public static FluidWrapper WHISKEY;
     public static FluidWrapper BEER;
     public static FluidWrapper RUM;
+    // Dye
+    public static FluidWrapper BLACK_DYE;
```

Comment:
Instead of this, we should mimic metal fluids, with a `Map<EnumDyeColor, FluidWrapper>`. I'm not concerned with making everything immutable as imo it makes more ugly code than it helps.

```java
// Storage
private static Map<EnumDyeColor, FluidWrapper> DYE;

public static FluidWrapper getFluidFromDye(EnumDyeColor color) { ... }
```